cogstat/cogstat.py

```diff
@@ -822,9 +822,18 @@ def explore_variable_pair(self, x, y, xlims=[None, None], ylims=[None, None]):
         sample_result = '<cs_h2>' + _('Sample properties') + '</cs_h2>'
         residual_title = None
         residual_graph = None
+        normality = None
+        assumptions_result = None
         if meas_lev == 'nom':
             sample_result += cs_stat.contingency_table(data, [x], [y], count=True, percent=True, margins=True)
         elif meas_lev == 'int':
+
+            # Test of multivariate normality
+            assumptions_result = '\n' + '<cs_h3>' + _('Checking assumptions of inferential methods') + '</cs_h3>\n'
+            assumptions_result += '<decision>' + _('Testing multivariate normality of variables') + '</decision>\n'
+            normality, norm_text = cs_hyp_test.multivariate_normality(self.data_frame[[x, y]], [x, y])
```

Comment:
```suggestion
            normality, norm_text = cs_hyp_test.multivariate_normality(data, [x, y])
```
Otherwise you'll keep the NaN values which is against your assumption in multivaritate_normality (at least the doc string says so).